src/util/scrollbar.ts

```diff
@@ -0,0 +1,44 @@
+import {Injectable, Inject} from '@angular/core';
+import {DOCUMENT} from '@angular/common';
+
+
+
+@Injectable()
+export class ScrollBar {
+  constructor(@Inject(DOCUMENT) private _document) {}
+
+  compensate() {
+    if (!this._isPresent()) {
+      return () => {};
+    }
+    const width = this._getWidth();
+    return this._adjustBody(width);
+  }
+
+  private _adjustBody(width: number) {
+    const {body} = this._document;
```

Comment:
Wouldn't explicit `const body = this._document.body;` be more readable here?

Reply:
Same on line 33, if you want to do something with it. But it's just a question of style to me...

Reply:
Well, destructuring is standard feature, used elsewhere in this project, and avoids repetition. Then I could say it makes more sense when destructuring pure data objects (like parameters). I don't know 😛